test_container/tests/test/python3/all/sys_executable.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+from exasol_python_test_framework import udf
+
+
+class SysExecutableTest(udf.TestCase):
+    def setUp(self):
+        self.schema = "SYS_EXECUTABLE"
```

Comment:
```suggestion
        self.schema = "SYS_EXECUTABLE_TEST"
```

Reply:
As at least me as a noob mixed up database internals like `SYS.` tables with python built-ins like `sys.executable` maybe it's worth to make already the name of the database schema signal, that it's just a name without any special semantics.